src/components/FloatingActionButton/FabPlusIcon.js

```diff
@@ -1,34 +1,57 @@
+import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React, {useEffect} from 'react';
-import Animated, {Easing, interpolateColor, useAnimatedProps, useSharedValue, withTiming} from 'react-native-reanimated';
+import Animated, {createAnimatedPropAdapter, Easing, interpolateColor, processColor, useAnimatedProps, useSharedValue, withTiming} from 'react-native-reanimated';
 import Svg, {Path} from 'react-native-svg';
 import useTheme from '@hooks/useTheme';
 
 const AnimatedPath = Animated.createAnimatedComponent(Path);
 
+// Adapting fill and stroke properties from react-native-svg to be able to animate them with Reanimated
```

Comment:
NAB: i think this comment isn't needed. it doesn't explain `why` we're doing this

```suggestion
```